roles/rsyslog/tasks/main_core.yml

```diff
@@ -10,25 +10,38 @@
     - not __logging_is_ostree | d(false)
   block:
     # it is assumed that the only packages providing config files that might
-    # be modified are the base packages - if this is not so, then additional
-    # packages will need to be added to this check
+    # be modified are the rsyslog related base packages (but not others like iproute);
+    # if this is not so, then additional packages will need to be added to this check
     - name: Get status of rsyslog packages
       command: rpm -V {{ item }}  # noqa command-instead-of-module
-      loop: "{{ __rsyslog_base_packages }}"
+      loop: "{{ __rsyslog_base_packages | select('search', 'rsyslog') | list }}"
       register: __rsyslog_package_status
       failed_when: false
       changed_when: false
 
-    - name: Reset original confs - logging package is absent
-      package:
-        name: "{{ __rsyslog_base_packages }}"
+    - name: Clean up old rsyslog main configuration
+      file:
+        path: /etc/rsyslog.conf
+        state: absent
+
+    - name: Scan for old rsyslog configuration drop-ins
```

Comment:
The problem here is that other packages may provide /etc/rsyslog.d drop-in files.  For example, cloud-init.  We don't want to remove files provided by other packages, only files provided by the rsyslog* packages.

Reply:
This is a change from #440 and got addressed there.